config/locales/views/groups/en.yml

```diff
@@ -35,6 +35,7 @@ en:
         already_pending: 'Could not invite %{user_name}: this student already has a pending invitation for this group.'
         extension_exists: You cannot modify a group with an extension to the due date. Please contact your instructor if you wish to make changes.
         group_max_reached: 'Could not invite %{user_name}: group maximum has been reached.'
+        inactive_grader: Cannot assign inactive grader(s) '%{user_names}' due to grader inactivity.
```

Comment:
Delete the phrase "due to grader inactivity". It's fine to just end with the user names.

Also, omit the quotes around the grader names.